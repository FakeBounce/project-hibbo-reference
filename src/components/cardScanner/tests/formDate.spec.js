import React from 'react';
import renderer from 'react-test-renderer';

import FormDate from '../components/FormDate';

describe('FormDate', () => {
  const fn = jest.fn();
  const component = renderer.create(
    <FormDate
      labelMonth="Some label"
      valueMonth=""
      errorMonth={false}
      labelYear="Some label"
      valueYear=""
      errorYear={false}
      onChange={fn}
      validate={fn}
      maxLength={3}
    />,
  ).toJSON();

  it('should render the component', () => {
    expect(component).toMatchSnapshot();
  });
});
