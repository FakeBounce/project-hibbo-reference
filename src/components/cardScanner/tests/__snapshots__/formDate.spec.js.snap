// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormDate should render the component 1`] = `
<View
  style={undefined}
>
  <View
    style={
      Array [
        undefined,
        Object {
          "width": 50,
        },
      ]
    }
  >
    <TextInput
      autoCapitalize="words"
      autoCorrect={false}
      autoFocus={false}
      keyboardType="numeric"
      maxLength={3}
      onChangeText={[Function]}
      onSubmitEditing={[Function]}
      placeholder="Some label"
      placeholderTextColor="#979797"
      returnKeyType="next"
      style={
        Array [
          undefined,
          Object {
            "color": "#413D3D",
            "width": 50,
          },
        ]
      }
      underlineColorAndroid="transparent"
      value=""
    />
  </View>
  <View
    style={
      Array [
        undefined,
        Object {
          "width": 50,
        },
      ]
    }
  >
    <TextInput
      autoCapitalize="words"
      autoCorrect={false}
      autoFocus={false}
      keyboardType="numeric"
      maxLength={3}
      onChangeText={[Function]}
      onSubmitEditing={[Function]}
      placeholder="Some label"
      placeholderTextColor="#979797"
      returnKeyType="next"
      style={
        Array [
          undefined,
          Object {
            "color": "#413D3D",
            "width": 50,
          },
        ]
      }
      underlineColorAndroid="transparent"
      value=""
    />
  </View>
</View>
`;
